joomla/joomla-cms
# [4.0] Validation in Mass Mail Users

Pull Request for Issue #33450  

### Summary of Changes
form-validation added

### Testing Instructions
`Users` > `Mass Mail Users` > `Click Send Email`
Apply PR
npm i
Repeat and See the changes

### Actual result BEFORE applying this Pull Request
![before-mass](https://user-images.githubusercontent.com/8440661/116753128-47288000-aa0f-11eb-986d-c148a4401eed.png)

### Expected result AFTER applying this Pull Request
![user-mail-after](https://user-images.githubusercontent.com/61203226/116814239-7a851f00-ab75-11eb-9dd7-c6106579ddec.JPG)

### Documentation Changes Required
None
